docs/versioning.rst

```diff
@@ -0,0 +1,48 @@
+Versioning and public API
+=========================
+
+Acts uses `Semantic Versioning <https://semver.org/spec/v2.0.0.html>`_ to
+indicate breaking changes in its public API. A breaking change will result in a
+new major version. The `Conventional Commits
+<https://www.conventionalcommits.org/en/v1.0.0/>`_ convention is used for pull
+requests to track the type of changes that have been merged and ensure that the
+version number is increased correctly.
+
+Since Acts is still under active development not all visible symbols are
+automatically considered part of the public API and as such fall under the
+Semantic Versioning rules. Which symbols are part of the public API (and which
+ones are not) is outlined below.
```

Comment:
The subset of symbols that are currently considered part of the public API ...

Reply:
Fixed.